### PR TITLE
Refactor: cluster, dataset, infobox, layer, plugin and project gql query files

### PR DESCRIPTION
**Pull request by airslice**
close reearth/reearth#272

# Overview

Cleaning GraphQL queries and mutations following standards written in https://github.com/reearth/reearth/issues/272

This is the 2/3 PR of the series task.

This PR includes the cleaning of:
- cluster.ts
- dataset.ts
- infobox.ts
- layer.ts
- plugin.ts
- project.ts

## What I've done

- cluster:
  - PascalCase `removeCluster`

- dataset:
  - update query name `datasetSchemas` to `GetDatasetSchemas`
  - PascalCase`importGoogleSheetDataset`
  - reorder mutation exports
 
- infobox:
  - PascalCase `getBlocks` `createInfobox` `removeInfobox` `addInfoboxField` `moveInfoboxField` `removeInfoboxField`
  - reorder mutation exports

- layer:
  - PascalCase `moveLayer` `addLayerGroupFromDatasetSchema`
  - reorder mutation exports

- plugin:
  - update query name `InstallablePlugins` to `GetInstallablePlugins`
  - update query name `InstalledPlugins` to `GetInstalledPlugins`
  - PascalCase `uninstallPlugin`

- project:
  - update query name `Project` to `GetProjects`
  - PascalCase `updateProject` `deleteProject` `updateProjectAlias` `archiveProject` `updateProjectBasicAuth`
  - reorder mutation exports 

- update related hooks
- generate the gql api tsx

### CODE REVIEW

File: src/gql/queries/dataset.ts [hunk 1]
```diff
@@ -84,6 +90,33 @@ export const GET_LINKABLE_DATASETS = gql`
...
+  }
+`;
+
+export const REMOVE_DATASET_SCHEMA = gql`
+  mutation RemoveDatasetSchema($schemaId: ID!) {
+    removeDatasetSchema(input: { schemaId: $schemaId }) {
+      schemaId
+    }
+  }
+`;
```
Comment (KaWaite) [inline src/gql/queries/dataset.ts:118]: If you could, can you investigate these a little and see if you can remove one? If the backend for `removeDataseetSchema` has the `force` argument as optional, you could update here and use the one mutation for both (with and without force)!

File: src/gql/queries/dataset.ts [hunk 2]
```diff
@@ -167,7 +200,7 @@ export const SYNC_DATASET = gql`
 `;
 
 export const IMPORT_GOOGLE_SHEET_DATASET = gql`
-  mutation importGoogleSheetDataset(
+  mutation ImportGoogleSheetDataset(
```
Comment (KaWaite) [inline src/gql/queries/dataset.ts:203]: Can you make these the same as the backend's importDatasetFromGoogleSheet? If we ever add more import dataset types it'd be nice to have
ImportDatasetFromGoogleSheet
ImportDatasetFromXXXXX
ImportDatasetFromYYYYY

File: src/gql/queries/plugin.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { gql } from "@apollo/client";
 
 // import { layerFragment } from "@reearth/gql/fragments";
```
Comment (KaWaite) [inline src/gql/queries/plugin.ts:3]: If this isn't being used, please remove

**Review by KaWaite — COMMENTED** [review]
Please make the changes I asked for and then should be good to go!

**Comment by airslice** [comment]
got it. resolving now

File: src/gql/queries/dataset.ts (hunk 1 above)
Comment (airslice) [reply src/gql/queries/dataset.ts]: removed RemoveDatasetSchema as it seems no longer in use

File: src/gql/queries/dataset.ts (hunk 2 above)
Comment (airslice) [reply src/gql/queries/dataset.ts]: updated to ImportDatasetFromGoogleSheet

File: src/gql/queries/plugin.ts (hunk 3 above)
Comment (airslice) [reply src/gql/queries/plugin.ts]: removed

**Review by KaWaite — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**